installer/scripts/service_control

```diff
@@ -470,11 +470,9 @@ stop_omsagent()
 
 restart_omsagent()
 {
+    enable_omsagent_service
```

Comment:
added here because in any case, with previous setup it would either be called if `this_omsagent_running` returned true, or in `start_omsagent` when it is called.